repos/system_upgrade/el7toel8/actors/systemfacts/libraries/systemfacts.py

```diff
@@ -208,11 +208,14 @@ def _parse(r):
 def get_selinux_status():
     ''' Get SELinux status information '''
 
-    try:
-        import selinux
-    except ImportError:
-        api.report_error("SELinux Import Error", details="libselinux-python package must be installed.")
-        return
+    import sys
+    if not 'selinux' in sys.modules:
```

Comment:
what is this solving? I can't think of anything. If it is already imported, then it won't import it again anyway. So the question at hand here, is what does this solve?

Reply:
@vinzenz,  My intention: I want to ensure that when I mock selinux function in an unit test (PR #56) this mocking will not be overridden by local import in any way.
However, as I had dug deeper, it doesn't seem to be much helpful (in taking care of local imports in this way) as the module's name presence in sys.module doesn't actually mean it's visible in this function and then it takes as "local variable 'selinux' referenced before assignment". I'll rather remove it, then.

Reply:
Well you can do this if you need to know if it's already there:
```python
if not 'selinux' in globals():
...
```